server/src/main/resources/file.conf

```diff
@@ -1,4 +1,15 @@
 ## transaction log store, only used in seata-server
+server {
```

Comment:
I have some question about this pr.
1.The configuration ofNetty we already had the prefix name "transport.xxx.xxx", how about change to transport.xxx.xxx instead of server.xxx.xxx
2.How about move this configuration in the NettyServerConfig.java ?
3.We do not show the configuration of Netty in file.conf. How about remove it and add to
```
file.conf.example
script/server/file.conf
script/server/file.properties
script/server/file.yml
```

Reply:
+1

Reply:
1 and 2:
I think the thread pool configurations are related to the server itself instead of transport.
So the server.xxx.xxx is more better than transport.
If the configuration is set for client and server communication, it is better using transport.xxx.xxx

3. In fact, I can remove the default value in file.conf.
I put the default value into the file.conf, because it is the responsibility to reminder we can change the value for performance problems.

Reply:
Thanks for you replay. For now we already put the config of client in the NettyServerConfig and the server in the NettyServerConfig, and they begin with 'transport.xxx.xxx'. How do you think move the config of server to the NettyServerConfig and let it begin with 'transport.xxx.xxx'. May be you are right and I hope can keep in touch with after this. But for now I think the consistency is more important than the correctness.

Reply:
Now I put the configure into NettyServerConfig and change the name from server.xxx.xxx to transport.xxx.xxx